src/xtopcom/xvledger/src/xvblock.cpp

```diff
@@ -478,11 +480,16 @@ namespace top
             return xatomic_t::xadd(m_modified_count);
         }
         
-        void     xvqcert_t::reset_modified_count()
+        void     xvqcert_t::reset_modified_count() const
```

Comment:
why adding const while m_modified_count will be modified?

Reply:
get_hash_to_sign function is const.